python-scripts/xbox/interface/if_xbdm.py

```diff
@@ -77,28 +76,31 @@ def xbdm_command(cmd, length=None):
   return lines
 
 def GetModules():
-  return [] # FIXME: Make this work..
+  modulesList = []
   lines = xbdm_command("modules")
+
   for line in lines:
-    print(line)
-    chunks = line.split()
-    for chunk in chunks:
-      if (chunk[0:6] == b'name=\"'):
-        print("  name = '" + str(chunk[6:-1]) + "'")
-      elif (chunk[0:5] == b'base='):
-        print("  base = '" + str(chunk[5]) + "'")
-      else:
-        print('  unparsed: ' + str(chunk))
-        pass
-        #assert(False)
-  return []
-#202- multiline response follows
-#name="xboxkrnl.exe" base=0x80010000 size=0x001380e0 check=0x0013eb88 timestamp=0x3cfcc86a
-#name="xbdm.dll" base=0xb0011000 size=0x000620c0 check=0x0007007d timestamp=0x407c6068
-#name="vx.dxt" base=0xb00b3000 size=0x00018d80 check=0x00024302 timestamp=0x407c620b
-#name="wavebank.exe" base=0x00010c40 size=0x0004fa40 check=0x00000000 timestamp=0x4c14abd2 tls xbe
-#.
 
+    module ={}
+    line = line.decode("utf-8")
+
+    nameAddrChunks = line.split()
+
+    for nameAddrChunk in nameAddrChunks:
+       singleChunk = nameAddrChunk.split("=")    #name="xbdm.dll" -> ['name', '"xbdm.dll"']
```

Comment:
nit: spaces before comment seem unnecessary and don't add to readability, should also make clear that the given comment is only an `Example: `